google/cloud/forseti/services/client.py

```diff
@@ -69,6 +70,24 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def create_interceptors(endpoint):
+    """Create gRPC client interceptors.
+
+    Args:
+        endpoint (str): The gRPC server endpoint. E.g: 'localhost:443'
+
+    Returns:
+        tuple: A tuple of interceptors.
+    """
+    interceptors = []
+    if is_opencensus_enabled():
+        # It's okay for this to be enabled on the client, even if the tracing
+        # flag is disabled on the server.
+        from google.cloud.forseti.common.opencensus import tracing
```

Comment:
We have to make an exception here: if we put the import at the top of the file, the import would fail if tracing is disabled because `opencensus` is not even installed. 

That's why the code checks if `is_opencensus_enabled()` before running this import, otherwise we would fail with an `ImportError`. 

I find this more elegant than `try`/`except` blocks for imports in `server.py` and `client.py`, what do you think ?

Reply:
Sure, but keep in mind that this means wherever we'll need to add spans to traces (in each different service, such as inventory), we'll have to put the same try / except at the top of each file that we add tracing too.
Is this something acceptable ?